src/index.ts

```diff
@@ -232,11 +232,13 @@ function getRoute(route: Route, isReadonly: boolean): string {
 const getRoutesPrelude = `// DO NOT EDIT MANUALLY - metarpheus-generated
 import axios from 'axios'
 import { pathReporterFailure } from 'io-ts/lib/reporters/default'
+import * as t from 'io-ts'
+import * as m from 'model-ts'
```

Comment:
I'd make this more self contained by importing from '. /model'

Reply:
mmm... I'm not sure I understand. Do you mean `./model-ts`?